build/initialization.sh

```diff
@@ -80,6 +80,9 @@ ControlPort 9051
 CookieAuthentication 1
 EOF
 
+# enable embassyd dns server
+systemd-resolve --set-dns 172.18.0.1
```

Comment:
Are we installing the systemd-resolve on the pi along with the embassyd?

Reply:
I'm pretty sure it was already there.